tensorflow/core/kernels/tensor_array.h

```diff
@@ -35,7 +35,14 @@ limitations under the License.
 namespace tensorflow {
 
 typedef Eigen::ThreadPoolDevice CPUDevice;
+
+#if GOOGLE_CUDA
```

Comment:
is that wrapper needed for CUDA ? they do not use it for the GPUDevice typedef in general